internal/command/checks/list.go

```diff
@@ -0,0 +1,104 @@
+package checks
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/superfly/flyctl/api"
+	"github.com/superfly/flyctl/client"
```

Comment:
We're going to get rid for the presenters package once every command left in `cmd` is gone. You'd probably be better off adding what you need to the render package. Here is an example on how to render `AllocationStatuses`:
https://github.com/superfly/flyctl/blob/832c567731a1ea687a83af76191ff20e767eb23e/internal/render/allocations.go#L16

Reply:
addressed in 0e2b3f4c